### ISSUE TITLE
CSP issue with 15

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/issuetemplate#50]
**Browser console**

```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”).
issuetemplate:1:1
JQMIGRATE: Migrate is installed, version 1.4.0 core.js:7:542
Content Security Policy: The page’s settings blocked the loading of a resource at eval (“script-src”).
build.js:11190:11
XML Parsing Error: not well-formed
Location: https://192.168.100.36/core/fonts/Nunito-Regular.woff2
Line Number 1, Column 5: Nunito-Regular.woff2:1:5
Source map error: TypeError: NetworkError when attempting to fetch resource.
Resource URL: https://192.168.100.36/apps/issuetemplate/js/build/build.js?v=824ae22c-0
Source Map URL: build.js.map[Learn More]
```

_Originally posted by @enoch85 in https://github.com/nextcloud/issuetemplate/issues/49#issuecomment-445472824_

**Comment by e-alfred** [comment]
Is it possible to supply a new version to the App Store that supports Nextcloud 15? I think the issue template helps a lot to create useful bug reports so it should be available once again.

**Comment by juliushaertl** [comment]
CSP issues have been fixed with 0de9d83 I'll try to release a new version the next days.
